edit/lint.js

```diff
@@ -93,23 +93,23 @@ function updateLintReport(cm, delay) {
           const pos = isActiveLine ? 'cursor' : (info.from.line + ',' + info.from.ch);
           // stylelint rule added in parentheses at the end
           const rule = linter === 'stylelint' ?
-            info.message.substring(info.message.lastIndexOf('('), info.message.length) :
+            info.message.substring(info.message.lastIndexOf('('), info.message.length).replace(/[()]/g, '') :
             / at line \d.+$/;
```

Comment:
It seems all stylelint messages start with the obvious and noisy when repeated `Unexpected` word which I suggest stripping just like we strip the `at line` from csslint's output.